documentation/IDEasy-contribution-getting-started.adoc

```diff
@@ -142,7 +143,7 @@ Let's get to the practical part of the Bash-usage.
 **If you are using Windows**, make sure that you have git-bash installed, so you can execute the Bash commands mentioned in this introduction. [Here you can find git for windows](https://git-scm.com/download/win).
```

Comment:
```suggestion
**If you are using Windows**, make sure that you have git-bash installed, so you can execute the Bash commands mentioned in this introduction. (https://git-scm.com/download/win[Here you can find git for windows]).
```

Reply:
Yes, dont get messed with ugly MarkDown URL syntax that is required in github comments.
In AsciiDoc it is easy to remember:
https//some.url/path[some link text]
While in MarkDown you never remember which of the following 4 was the correct syntax:
```
[some link text](https//some.url/path)
(some link text)[https//some.url/path]
(https//some.url/path)[some link text]
[https//some.url/path](some link text)
```

Rule of thumb: AsciiDoc is the much better MarkDown ;)

Reply:
Another thought: Why do we repeat the installation guide at all in getting-started guide?
We already have this:
https://github.com/devonfw/IDEasy/blob/main/README.adoc
and the more detailed variant here:
https://github.com/devonfw/IDEasy/blob/main/documentation/setup.adoc

IMHO we should not explain this here again. If we think something like this is needed here, it should be sufficient to link `setup.adoc` from here instead.